src/Components/PhoneNumberInput/usePhoneNumberValidation.ts

```diff
@@ -0,0 +1,90 @@
+import { fetchQuery, graphql } from "react-relay"
+import { useSystemContext } from "System"
+import { usePhoneNumberValidationQuery } from "__generated__/usePhoneNumberValidationQuery.graphql"
+
+export interface PhoneNumberValidationResult {
+  // TODO: Why are these nullable?
+  international: string | null
+  isValid: boolean | null
+  national: string | null
+  originalNumber: string | null
+  regionCode: string
+}
+
+export type PhoneNumber = {
+  national: string
+  regionCode: string
+}
+
+interface UsePhoneNumberValidation {
+  onValidation(result: PhoneNumberValidationResult): void
+  phoneNumber: PhoneNumber
+}
+
+export const usePhoneNumberValidation = ({
+  onValidation,
+  phoneNumber,
+}: UsePhoneNumberValidation) => {
+  const { relayEnvironment } = useSystemContext()
+
+  const handleValidation = async () => {
+    if (phoneNumber.national.length < 5) {
+      onValidation({
+        international: null,
+        isValid: false,
+        national: phoneNumber.national,
+        originalNumber: phoneNumber.national,
+        regionCode: phoneNumber.regionCode,
+      })
+
+      return
+    }
+
+    if (!relayEnvironment) return
```

Comment:
I think this is safe to remove

Reply:
I mean it _is_ but our types don't say it is. It's because there's no defaults set in the `SystemContext` — at some point I need to go through and get rid of all of those optionals.